src/openfermion/transforms/_jordan_wigner.py

```diff
@@ -61,9 +65,45 @@ def jordan_wigner(operator):
                     z_factors + ((ladder_operator[0], 'Y'),), 0.5j)
             transformed_term *= pauli_x_component + pauli_y_component
         transformed_operator += transformed_term
+
     return transformed_operator
 
 
+def jordan_wigner_diagonal_coulomb_hamiltonian(operator):
```

Comment:
Perhaps this should be called by the standard jordan_wigner function when it is passed a DiagonalCoulombHamiltonian. That way one does not need to keep track of multiple different jordan_wigner functions.

Reply:
I did do it that way. However, this reminds me that I should also update `get_sparse_operator`.